packages/cli-kit/src/private/node/ui/components/ConcurrentOutput.tsx

```diff
@@ -71,49 +77,95 @@ function currentTime() {
  */
 const ConcurrentOutput: FunctionComponent<ConcurrentOutputProps> = ({
   processes,
+  prefixColumnSize = 12,
   abortSignal,
   showTimestamps = true,
   keepRunningAfterProcessesResolve = false,
 }) => {
   const [processOutput, setProcessOutput] = useState<Chunk[]>([])
   const {exit: unmountInk} = useApp()
-  const prefixColumnSize = Math.max(...processes.map((process) => process.prefix.length))
-  const concurrentColors: TextProps['color'][] = useMemo(() => ['yellow', 'cyan', 'magenta', 'green', 'blue'], [])
-  const lineColor = useCallback(
-    (index: number) => {
-      const colorIndex = index < concurrentColors.length ? index : index % concurrentColors.length
-      return concurrentColors[colorIndex]!
-    },
-    [concurrentColors],
-  )
-  const writableStream = useCallback(
```

Comment:
@isaacroldan The use of `useCallback` here seemed unnecessary and I ended up removing it while I was iterating on state management for the seen prefixes. I could probably reintroduce it now, but is it actually needed?

Reply:
Linting encouraged me to add it back... I believe that using it improves rendering performance.